include/irr/asset/ICPUPipelineCache.h

```diff
@@ -110,10 +110,21 @@ class ICPUPipelineCache final : public IAsset
 	size_t conservativeSizeEstimate() const override { return 0ull; /*TODO*/ }
 	void convertToDummyObject(uint32_t referenceLevelsBelowToConvert = 0u) override
 	{
-		if (canBeConvertedToDummy())
+		if (!isADummyObjectForCache() && canBeConvertedToDummy())
 			m_cache.clear();
 	}
 
+	bool restore(IAsset* _other) override
+	{
+		if (!IAsset::restore(_other))
+			return false;
+
+		auto* other = static_cast<ICPUPipelineCache*>(_other);
+		std::swap(m_cache, other->m_cache);
```

Comment:
can we even restore this?

Reply:
hm.. good point. This asset is kinda special because the idea was GPU object->asset->potentially save on disk

Reply:
we'd have to check a few things before restoring, then I guess?